raiden/ui/cli.py

```diff
@@ -412,14 +411,17 @@ def options(func):
                 "--proportional-fee",
                 help="Mediation fee as ratio of mediated amount in parts-per-million (10^-6).",
                 default=DEFAULT_MEDIATION_PROPORTIONAL_FEE,
-                type=int,
+                type=click.IntRange(min=0),
```

Comment:
Also set `max=10**6`?

Reply:
Should we limit this? Why not allow 2*10**18?